lib/Cookie.php

```diff
@@ -172,6 +175,24 @@ public function isHttpOnly()
         return $this->offsetGet('httpOnly');
     }
 
+    /**
+     * The cookie's same-site value.
+     *
+     * @param string $sameSite
```

Comment:
Maybe should propose null too to redet the value?

Reply:
All setters don't specifically define that `null` could be set. For example, the setter for "setHttpOnly" only "allowes" boolean values, but "isHttpOnly" is defined as returning boolean and `null`. However I don't have a strong opinion about adding the additional `null` param type.